api/test/integration/test_rbac_white_logtest_endpoints.tavern.yaml

```diff
@@ -0,0 +1,41 @@
+---
+test_name: PUT /logtest and DELETE /logtest/sessions/{token}
```

Comment:
Separate tests for different endpoints (useful for documentation and reporting)

Reply:
Tests separated and new `PUT /logtest` test case added.